docs: Introdução e Controle com Streaming

julia-lika opened the pull request.

zzaved reviewed (approved):
Muito bom, Isa! Parabéns.

daviiabreu committed:
feat: criei uma pasta

zzaved committed:
feature: adiciona padding e redução

Este commit adiciona o padding e a redução do tamanho da imagem

daviiabreu committed:
feat: adiciona as imagens de fissuras

daviiabreu committed:
feat: adiciona as imagens da fissura

iisabelledantas committed:
feat: adiciona o componente da  mini galeria

Lucas-nepomuceno committed:
Merge pull request #6 from Inteli-College/feat/cria-tela-upload

feat: adiciona o componente da  mini galeria

Lucas-nepomuceno committed:
Merge pull request #5 from Inteli-College/feat/tela-principal

Feat/tela principal

zzaved committed:
feature: adiciona testes

Este commit adiciona os testes entre Sobel, Canny e Ann. A comparação dos trÊs é importante para compreender qual aplicar de fato no refinamento das imagens.

zzaved committed:
feature: adiciona código de comparação de filtros

Este commit adiciona o código que busca comparar Canny, Sobel e ANN na aplicação de imagens

zzaved committed:
Merge branch 'feat/ia-agro-lei' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/ia-agro-lei

daviiabreu committed:
feat: preprocess visualization

iisabelledantas committed:
fix: adiciona a funcionalidade de ampliar e excluir fotos na  mini galeria

daviiabreu committed:
feat: folder organization and pipeline base extructure

zzaved committed:
feature: adiciona pipeline para estudos

Conforme combinado com o meu colega de grupo, Davi, iremos modularizar filtros e criar uma pipeline para essa aplicação. Nese commit, estou estudando como funciona a modularização de filtros e criação de uma pipeline

Lucas-nepomuceno committed:
refactor: reorganiza as pastas

Lucas-nepomuceno committed:
Merge branch 'feat/cria-tela-upload' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/upload-mini-galeria

Lucas-nepomuceno committed:
feat: adiciona a tela de upload com a mini galeria

zzaved committed:
feature: adiciona modularização de filtros

Este arquivo adiciona a modularização dos filtros a serem aplicados na imagem no pré-processamento na pipeline

zzaved committed:
feature: adiciona vetorização do padding

Este commit adiciona a vetorização do padding nas imagens para serem adicionadas na pipeline

zzaved committed:
Feature: adiciona vetorização

Este commit adiciona a vetorização de componentes para a pipeline e uma pipeline de testes

Lucas-nepomuceno committed:
fix: coloca as paginas como paginas

iisabelledantas committed:
Merge pull request #8 from Inteli-College/feat/upload-mini-galeria

Feat/upload mini galeria

iisabelledantas committed:
feat: adiciona a base do front-end da tela de historico

lupetenazzi committed:
feat: adiciona a função que realiza a busca das imagens no supabase com a coluna app igual a true

esse commit adiciona a função e adiciona uma tentativa falha de teste devido a um erro encontrado em uma outra pagina, para proximos passos a função deve ser alterada de maneira que ela nao execute automaticamente e sim ao ser selecionada por um botão

daviiabreu committed:
feat: pipeline completa com augmentation e filtros integrados com pablo

daviiabreu committed:
Merge pull request #9 from Inteli-College/feat/ia-agro-lei

Feat/ia agro lei

lupetenazzi committed:
fix: arruma funcao de buscar imagens

Lucas-nepomuceno committed:
feat: adiciona tanto a possibilidade de baixar relatorio quanto preview

daviiabreu committed:
feat: ai model swin transformer

lupetenazzi committed:
feat: adiciona o funcionamento do botao e da mini galeria na funcao de carregar imagens pelo servidor

Lucas-nepomuceno committed:
feat: cria a pagina que mostra um preview do relatorio

Lucas-nepomuceno committed:
Merge pull request #10 from Inteli-College/feat/funcao-image

Feat/funcao image

daviiabreu committed:
fix: erros no modelo corrigidos

daviiabreu committed:
fix: linux paths for supercomputer

daviiabreu committed:
fix: import base dir fix

daviiabreu committed:
fix: torch weights fix

daviiabreu committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/ai-model-davi

daviiabreu committed:
feat: modelos treinados

iisabelledantas committed:
feat: adiciona a tela de historico

iisabelledantas committed:
Merge pull request #11 from Inteli-College/feat/preview-historico

Feat/preview historico

daviiabreu committed:
feat: modulariza dentro de src para diferentes modelos em pastas diferentes

Lucas-nepomuceno committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/cria-tela-historico

lupetenazzi committed:
Merge pull request #13 from Inteli-College/feat/ai-model-davi

Feat/ai model davi

Lucas-nepomuceno committed:
feat: adiciona a pagina no botao

Lucas-nepomuceno committed:
Merge pull request #12 from Inteli-College/feat/cria-tela-historico

Feat/cria tela historico

iisabelledantas committed:
fix: corrige falhas no upload da imagem

daviiabreu committed:
feat: integracao da ia do pablo

zzaved committed:
Merge pull request #14 from Inteli-College/feat/integra-ias

feat: integracao da ia do pablo

daviiabreu committed:
feat: interference unificado

daviiabreu committed:
feat: trocando o swin por padrao

zzaved committed:
docs: adiciona estruturação de pastas

Este commit adiciona a estruturação de pastas para serem preenchidas na documentação

Melomm committed:
Merge pull request #16 from Inteli-College/docs/organiza-documentação

docs: adiciona estruturação de pastas

Lucas-nepomuceno committed:
Merge pull request #15 from Inteli-College/feat/integracao-ia-e-front

feat: interference unificado

Melomm committed:
Hotfix: Docs template

Melomm committed:
docs: Introdução e Controle com Streaming
